### ISSUE TITLE
Add CountDownLatch to keep track of `user.rating` requests.

### ISSUE THREAD

**Issue by evgenii-kanivets** [evgenii-kanivets/codeforces_watcher#84]
Now only `user.info` request is used to show `swipe-to-refresh`. It can be tricky because there might be a lot of `user.rating` requests that are not indicated. And also user can use `swipe-to-refresh` once more while first one isn't finished yet.

To solve this problem it's needed to hide `swipe-to-refresh` only after all requests are finished. For this purpose `CountDownLatch` can be used. Here is detailed [documentation](https://developer.android.com/reference/java/util/concurrent/CountDownLatch).

Basically you need to count number of `user.rating` requests that are needed, create `CountDownLatch` and then use it to hide `swipe-to-refresh`. It can be tricky because of threading stuff. Google it or read [this](https://beginnersbook.com/2013/03/java-threads/).

> HINT: You can place `await()` into another thread and then call `runOnUiThread` when it passed.
